DBZ-8634 Document Milvus sink (#6268)
debezium/debezium: documentation/modules/ROOT/pages/operations/debezium-server.adoc

So this string will be used as value for records without a payload.

|===

==== Milvus

Milvus is an open-source vector database designed for similarity search and retrieval of high-dimensional data, like embeddings from machine learning models (e.g. text, image, audio).
It is possible to either use to to process vector datatypes captured from a source database or use a transformation to calculate vector out of message field(s) and use it as an embedding.

Milvus sink taks the incoming message and upserts the `after` part into a collection.

> The collection cannot contain dots so the sink replaces all dots with underscore characters.

Author: ```suggestion
The collection cannot contain dots, so the sink replaces all dots with underscore characters.
```

[cols="35%a,10%a,55%a",options="header"]
|===
|Property
|Default
|Description

|[[milvus-type]]<<milvus-type, `debezium.sink.type`>>
|
|Must be set to `milvus`.

|[[milvus-uri]]<<milvus-uri, `debezium.sink.milvus.uri`>>
|_http://localhost:19530_
|(Optional) A URL to access Milvus database instance.

|[[milvus-database]]<<milvus-database, `debezium.sink.milvus.database`>>
| _default_
|(Optional) The name of the database containing target collections.

|===

===== Injection points

The Milvus sink behaviour can be modified by a custom logic providing alternative implementations for specific functionalities.
When the alternative implementations are not available then the default ones are used.

[cols="35%a,10%a,55%a",options="header"]
|===
|Interface
|CDI classifier
|Description

|[[milvus-ext-client]]<<milvus-ext-client, `io.milvus.v2.client.MilvusClientV2.MilvusClientV2`>>
|`@CustomConsumerBuilder`
|Custom configured instance of a `MilvusClientV2` used to access target collections.

|[[milvus-ext-stream-name-mapper]]<<milvus-ext-stream-name-mapper, `io.debezium.server.StreamNameMapper`>>
|
|Custom implementation maps the planned destination (topic) name into a Milvus collection.

> By default the name with dots replaced with underscores is used.

Reviewer: ```suggestion
By default, dots in a name are replaced with underscores.
```
